src/user-status/UserStatusPicker.js

```diff
@@ -0,0 +1,43 @@
+/* @flow strict-local */
+import React, { PureComponent } from 'react';
+import { FlatList } from 'react-native';
+
+import type { GetText } from '../types';
+import { TranslationContext } from '../boot/TranslationProvider';
+import { OptionButton } from '../common';
+import statusSuggestions from './userStatusTextSuggestions';
+
+type Props = {|
+  onPress: (status: string) => void,
+|};
+
+export default class UserStatusPicker extends PureComponent<Props> {
+  static contextType = TranslationContext;
+  context: GetText;
+
+  getTranslatedSuggestions = () =>
```

Comment:
Option button already uses `Label` so there won't be any problem. https://github.com/zulip/zulip-mobile/blob/master/src/common/OptionButton.js

It will get translated automatically, just u need to put them in message_en.json

Reply:
@jainkuniya I tried it, and while the option text shown on the screen is translated, the value of this option in `UserStatusScreen` is still in English, so when we send an update request to Zulip, the status is still in English.